mk-files/docs.mk

```diff
@@ -81,6 +76,17 @@ release-docs:
 		git commit -am "[ci skip] chore: update settings.sh and conf.py due to $(CLEAN_VERSION) release" && \
 		$(call dry-run,git push origin release-docs-$(CLEAN_VERSION)) && \
 		$(call dry-run,gh pr create --base master --title "Release docs for v$(CLEAN_VERSION)" --body ""); \
+	else \
+		git checkout -b release-docs-$(STAGING_BRANCH_DASH) origin/$(STAGING_BRANCH) && \
+		git checkout origin/master -- command-reference release-notes.rst && \
```

Comment:
Can we just `cp` these over, like we do the first time they're added?

Reply:
I think I was going to try this based on this comment, but on second thought I think this is simpler. We would have to repeat the process of cloning CLI to a temp folder and generating the command reference. Since the first docs step already does this and updates origin master, we can just pull it from there.

Reply:
No worries, we're going to switch to a docs team supported pipeline soon anyway!